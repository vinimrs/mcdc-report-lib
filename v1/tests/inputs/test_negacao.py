def decision(x, y):
  if not (x > 5 and y < 10):
      return "OK"
  return "NOT_OK"