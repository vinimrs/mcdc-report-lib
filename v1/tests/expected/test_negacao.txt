Relatório de Testes MC/DC
==============================

Decisão: if not (x > 5 and y < 10)
Condições: x > 5, y < 10

Casos de Teste MC/DC:
--------------------
Teste 1: x > 5=False | y < 10=True  | Resultado: True
Teste 2: x > 5=True  | y < 10=False | Resultado: True
Teste 3: x > 5=True  | y < 10=True  | Resultado: False

==============================
